it/controllers/employment/CheckYourBenefitsControllerISpec.scala

```diff
@@ -1038,6 +1048,173 @@ class CheckYourBenefitsControllerISpec extends IntegrationTest with ViewHelpers
           welshToggleCheck(user.isWelsh)
         }
 
+        "return a fully populated page without links when EOY and prior data exists" which {
```

Comment:
it might be worth specifying that there's missing employment details data in the test name to give some more context to the scenario

Reply:
fixed.